test/models/test_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
It's empty + you have rspec in your gemfile, so use it

Reply:
deleted